ui-v2/src/api/flow-runs/index.ts

```diff
@@ -76,6 +82,40 @@ export const buildListFlowRunsQuery = (
 	});
 };
 
+/**
+ * Builds a query configuration for fetching filtered flow runs
+ *
+ * @param filter - Filter parameters for the flow runs pagination query.
+ * @returns Query configuration object for use with TanStack Query
+ *
+ * @example
+ * ```ts
+ * const { data } = useSuspenseQuery(buildPaginateFlowRunsQuery());
+ * ```
+ */
+export const buildPaginateFlowRunsQuery = (
+	filter: FlowRunsPaginateFilter = {
+		page: 1,
+		sort: "ID_DESC",
```

Comment:
probably want to default to `START_TIME_DESC` rather than `ID_DESC`
```suggestion
		sort: "START_TIME_DESC",
```